src/index.ts

```diff
@@ -33,6 +36,7 @@ export type SerializationOptions = {
    * More details here: https://buildkite.com/docs/pipelines/dependencies#defining-explicit-dependencies
    */
   explicitDependencies?: boolean;
+  mutator?: MutatorFn;
```

Comment:
- [ ] add JSDOC here, explain what it does and link to the example

Reply:
Added a jsdoc, have a look

Reply:
you seem to have added them to the overall options type - if you put them right above the property (like we do for `explicitDependencies`) it will be attached to that particular property (and show up in editors correctly)

Reply:
done